src/common/meta/src/rpc/ddl.rs

```diff
@@ -932,6 +943,85 @@ impl TryFrom<DropDatabaseTask> for PbDropDatabaseTask {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct AlterDatabaseTask {
+    pub alter_expr: AlterDatabaseExpr,
+}
```

Comment:
It's better to convert the generated structure into our defined structure.

Reply:
`AlterTableTask` has the same problem, so I'd like to left a TODO here and refactor both in future prs.